lib/config-generator.js

```diff
@@ -400,6 +401,14 @@ class ConfigGenerator {
             }));
         }
 
+        if (this.webpackConfig.useImageWebpackLoader) {
+            rules.push(applyRuleConfigurationCallback('image-webpack', {
+                test: /\.(gif|png|jpe?g|svg)$/i,
+                loader: 'image-webpack-loader',
+                options: imageWebpackLoaderUtil.getOptions(this.webpackConfig)
+            }));
+        }
```

Comment:
The new loader should probably be part of the existing "images" rule:

https://github.com/symfony/webpack-encore/blob/53df05e04f87045f7fbe064b7802f49e26feaff9/lib/config-generator.js#L302-L306

See https://webpack.js.org/concepts/loaders/#configuration for how loaders are evaluted.